bitnami/metallb/templates/controller/deployment.yaml

```diff
@@ -144,6 +157,11 @@ spec:
         {{- if .Values.controller.sidecars }}
         {{- include "common.tplvalues.render" ( dict "value" .Values.controller.sidecars "context" $) | nindent 8 }}
         {{- end }}
+      volumes:
+        - name: cert
+          secret:
+            defaultMode: 420
+            secretName: webhook-server-cert
       {{- if .Values.controller.extraVolumes }}
-      volumes: {{- include "common.tplvalues.render" (dict "value" .Values.controller.extraVolumes "context" $) | nindent 8 }}
+      {{- include "common.tplvalues.render" (dict "value" .Values.controller.extraVolumes "context" $) | nindent 8 }}
```

Comment:
It would be nice if you could indent this to be aligned with the other elements in the array.